content/v2.0/get-started.md

```diff
@@ -218,7 +218,14 @@ The instructions below use Minikube, but the steps should be similar in any Kube
     kubectl apply -f https://raw.githubusercontent.com/influxdata/docs-v2/master/static/downloads/influxdb-k8-minikube.yaml
     ```
 
-    (**Note:** Always inspect YAML manifests before running `kubectl apply -f <url>`!)
+    <div class="warn block">
+      <p>
+        Always inspect YAML manifests before running <code>kubectl apply -f &lt;url&gt;</code>!
+      </p>
+    </div>
```

Comment:
I'm guessing you did this because the "nested" shortcode was breaking the layout. Shortcodes create a new "context" that don't know whether or not they're nested. They honor all indentation as if it was at the root level, so to get around it, you keep the shortcode tags indented, but all the contents of the shortcode should not be indented:

```
    {{% warn %}}
Always inspect YAML manifests before running `kubectl apply -f <url>`.
    {{% /warn %}}
```

Reply:
For me this methods breaks both the indentation and subsequent paragaph.
<img width="817" alt="Screen Shot 2020-01-09 at 2 53 08 PM" src="https://user-images.githubusercontent.com/19642016/72111544-f0fe6d80-32ef-11ea-976a-3b7f5d5d3ce1.png">

Reply:
Ok, yeah, I've tested it too. Definitely a bug with Hugo's markdown/shortcode renderer. I guess the raw HTML works for now until we can figure out the issue with the shortcode.